test/validate/fuel_ferc1_test.py

```diff
@@ -37,7 +37,11 @@ def test_fuel_ferc1_trivial(pudl_out_ferc1):
             id="oil_cost_per_unit",
             marks=pytest.mark.xfail(reason="FERC 1 fuel unit errors?"),
         ),
-        pytest.param(pv.fuel_ferc1_gas_cost_per_unit_bounds, id="gas_cost_per_unit"),
+        pytest.param(
```

Comment:
Can you put a comment in here with the full URL for the issue? Then it's clickable on GitHub.